gradle.properties

```diff
@@ -22,3 +22,4 @@ kotlin.js.compiler=ir
 
 kotlin.mpp.stability.nowarn=true
 kotlin.native.ignoreDisabledTargets=true
+kotlin.native.cacheKind=none
```

Comment:
Was this maybe just a stale cache? Would wiping the caches and reenabling this work?

Reply:
Let's try it out